### DISCUSSION TITLE
New features suggestion

### DISCUSSION THREAD

**Discussion by adesai1000** [ciderapp/Cider#919] [Feature Request]
Some of the new features I would like to have in the Cider App are:- 
1. Spacebar pauses music rather than pausing and scrolling down the page.
2. Show an Album's playtime.
3. Option to remove Tabs like Podcasts and Albums and just display playlists.
4. I'm also having issues with the app displaying lyrics.
5. Ability to arrange songs in a playlist.

**Comment by KillerParticles** [comment]
- Number 3 is already a feature, go to settings > General > Customise Sidebar Items
- Number 4, do you have Musixmatch lyrics turned on? Apple's lyrics no longer work
- Number 5 is already a feature too, just drag the songs up and down the playlist same to iOS, iTunes etc. Unless you mean something else?

**Reply by adesai1000** [reply]
![image](https://user-images.githubusercontent.com/68861149/164887477-ffe0c82a-99e8-4e46-ac02-66f556aaded8.png)

- I do not have number 3 on my app, even updating shows error updating cider.
- yes, Musixmatch is on yet I get the error
- and I did not know about dragging songs, thanks for that!

**Reply by KillerParticles** [reply]
The update button doesn't work from what I know, you can download the newest update from here https://github.com/ciderapp/cider-releases/releases

it does look like you are quite behind on your version so this should fix it. As for the lyrics, see if this newer version fixes it.

**Reply by adesai1000** [reply]
Hey I did update to the new version and I was able to edit the sidebar but I wasn't able to get lyrics working for some songs and it does work for some songs.

**Reply by yazninja** [reply]
> Hey I did update to the new version and I was able to edit the sidebar but I wasn't able to get lyrics working for some songs and it does work for some songs.

Apple has locked down their lyrics in the API, thus Cider cannot show lyrics from Apple's database, latest builds have musixmatch automatically on but this may not include the songs that were previously available before.

You can help add lyrics to musixmatch [here](https://www.musixmatch.com/contribute)

**Comment by yazninja** [comment]
> 1. Spacebar pauses music rather than pausing and scrolling down the page.

Added a Fix in develop builds version 1845 or hither should work
